rotkehlchen/exchanges/bybit.py

```diff
@@ -88,13 +81,11 @@ def bybit_symbol_to_base_quote(
     if split_symbol is not None and len(split_symbol) == 2:
         base_asset = asset_from_bybit(split_symbol[0])
         quote_asset = asset_from_bybit(split_symbol[1])
+    # If more four-letter quote suffixes are discovered, update the tuple below accordingly.
+    elif len(symbol) >= 4 and symbol[-4:] in ('USDT', 'USDC', 'USDE'):
```

Comment:
why not read the from the mappings? I believe is better than hardcoding them since it allows the user to fix issues without us needed to make a new release. something like what `six_letter_assets` was doing before.